napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -89,36 +89,54 @@ def _create_process(
         installer: InstallerTypes = "pip",
     ):
         process = QProcess()
-        if installer != "pip":
-            process.setProgram(installer)
-        else:
-            process.setProgram(self._sys_executable_or_bundled_python())
-
         process.setProcessChannelMode(QProcess.MergedChannels)
         process.readyReadStandardOutput.connect(
             lambda process=process: self._on_stdout_ready(process)
         )
+        env = QProcessEnvironment.systemEnvironment()
```

Comment:
Note we start with the system environment now, not a blank one!

Reply:
👍🏼